dockerfiles/alpine/obam-uk/Dockerfile

```diff
@@ -0,0 +1,116 @@
+# ----------------------------------------------------------------------------------------
```

Comment:
Since we have not tested uk toolkit for alpine images, lets not add them

Reply:
done https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc